src/routes/admin/ciraconfig/editCiraConfig.ts

```diff
@@ -57,12 +57,24 @@ export async function editCiraConfig (req, res): Promise<void> {
   }
 }
 
-function getUpdatedData (newConfig: any, oldConfig: CIRAConfig): CIRAConfig {
-  const config: CIRAConfig = { configName: newConfig.configName } as CIRAConfig
+const handleGenerateRandomPassword = (config: CIRAConfig, newConfig: CIRAConfig, oldConfig: CIRAConfig): CIRAConfig => {
```

Comment:
can we eliminate the need for three configs passed into this? Can we just use the new config/old config?

Reply:
done